tools/phenomenal/ms/show_chromatogram.xml

```diff
@@ -1,14 +1,20 @@
 <?xml version='1.0' encoding='UTF-8'?>
 <!--Proposed Tool Section: [Statistics]-->
-<tool id="show_chromatogram" name="show_chromatogram" version="1.1">
+<tool id="show_chromatogram" name="show_chromatogram" version="1.2">
 <!--
   <requirements>
-    <container type="docker">phnmnl/xcms</container>
+    <container type="docker">container-registry.phenomenal-h2020.eu/phnmnl/xcms</container>
   </requirements>
 -->
   <description>Shows the chromatogram of a single mzML file.</description>
   <command><![CDATA[
-    show_chromatogram.r $infile $outfile 2>&1
+if [[ "\$(head -n 2 $infile | tail -n 1 | grep mzML)" != "" ]];
+then filename="\$(head -n 40 $infile | grep "sample id" | sed -e "s/\"\>.*//" | sed -e "s/.*name\=\"//")";
+    xcmsfile="/tmp/\$filename.mzML";
+    cp $infile \$xcmsfile;
+    show_chromatogram.r \$xcmsfile $outfile 2>&1;
+else echo "Error! Input file is not mzML.";
```

Comment:
And exit with non-zero please as mentioned above. Thanks!

Reply:
Instead of doing a `cp`, I would prefer to rename the inputfile, thus, doing a `mv $infile $xcmsfile` in the Galaxy directory, but I do not know if it is allowed to change the name of the inputfile and whether Galaxy does like that and whether the input directory is writable for running jobs or not. @pcm32 : What is the preferred way in PhenoMeNal? I don't think we have any conventions on this yet?